Unit-Tests/JSON_Tests.m

```diff
@@ -29,6 +29,10 @@ - (void) test_CBLJSON_Date {
     XCTAssertEqualWithAccuracy([CBLJSON absoluteTimeWithJSONObject: @"2013-04-01T20:42:33Z"], 386541753.000, 1e-6);
     NSDate* date = [CBLJSON dateWithJSONObject: @"2013-04-01T20:42:33Z"];
     AssertEq(date.timeIntervalSinceReferenceDate, 386541753.000);
+    date = [CBLJSON dateWithJSONObject: @"2013-04-01T20:42:33+0000"];
+    AssertEq(date.timeIntervalSinceReferenceDate, 386541753.000);
```

Comment:
Unfortunately the patch isn't working for me: This assertion fails, because the date string failed to parse.